Makefile

```diff
@@ -1,36 +1,44 @@
-.PHONY: setup rebuild start superuser migrations migrate pull-production-data pull-production-media pull-staging-data pull-staging-media
-
-setup:
-	make rebuild
+.PHONY: setup
+setup: rebuild
```

Comment:
my preference would be to use something like https://github.com/wagtail/wagtail-localize/blob/main/Makefile whereby running `make` lists the available commands so you can get the list of available commands


aside: is it time to switch to using `docker compose` instead of `docker-compose` ?

Reply:
That'd be nice, but perhaps outside the scope here. I'd prefer to leave it as `docker-compose` for now. This isn't making use of any newer features that necessitate plugin rather than the standalone and people on the newer version already can use `compose-switch` or similar: https://github.com/docker/compose-switch